### PR TITLE
[OSDOCS-3989] OCP content port to ROSA and OSD: Nodes

### PR DESCRIPTION
**Pull request by mburke5678**
This PR ports the Nodes book from OCP to the OSD and ROSA books using single-sourcing.

Version(s):

    enterprise-4.13

Previews:
OCP: [Overview of nodes](https://62147--docspreview.netlify.app/openshift-enterprise/latest/nodes/)
ROSA: [Overview of nodes](https://62147--docspreview.netlify.app/openshift-rosa/latest/nodes/)
OSD: [Overview of nodes](https://62147--docspreview.netlify.app/openshift-dedicated/latest/nodes/)

Issue:
https://issues.redhat.com/browse/OSDOCS-3989

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by mburke5678** [comment]
@xueli181114 Can I please get a QE review of this PR? Thanks so much! This is my first PR in this effort. If it doesn't meet your expectations, please let me know.

**Comment by mburke5678** [comment]
@xueli181114 Can I please get a QE review of this PR? Thanks so much! This is my first PR in this effort. If it doesn't meet your expectations, please let me know.

**Comment by mburke5678** [comment]
Ying Zhang [LGTM in Jira](https://issues.redhat.com/browse/OSDOCS-3989?focusedId=22871991&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-22871991).

**Comment by lyman9966** [comment]
@lihongan Can you help to review the chapter “Remote worker nodes on the network edge” in https://62147--docspreview.netlify.app/openshift-enterprise/latest/nodes/edge/nodes-edge-remote-workers? Thanks!

**Comment by lyman9966** [comment]
@kasturinarra Can you help to review the chapter “Controlling pod placement onto nodes (scheduling)” in https://62147--docspreview.netlify.app/openshift-enterprise/latest/nodes/scheduling/nodes-scheduler-about ? Thanks

**Comment by kasturinarra** [comment]
> @kasturinarra Can you help to review the chapter “Controlling pod placement onto nodes (scheduling)” in https://62147--docspreview.netlify.app/openshift-enterprise/latest/nodes/scheduling/nodes-scheduler-about ? Thanks

sure, will do.

**Comment by kasturinarra** [comment]
section `Controlling pod placement using the scheduler` looks good to me and i understand that there is no change from what is written in ocp doc . If not please let me know.

**Comment by karanthshashank** [comment]
@lihongan could you please help review this PR we have got a lgtm from @kasturinarra for the Controlling pod placement part. Let us know if any updates or changes needs to be made.

**Comment by mburke5678** [comment]
Ying Zhang [lgtm in Jira](https://issues.redhat.com/browse/OSDOCS-3989?focusedId=23232312&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-23232312).

**Comment by mburke5678** [comment]
@lihongan Can you help to review the chapter “Remote worker nodes on the network edge” in https://62147--docspreview.netlify.app/openshift-enterprise/latest/nodes/edge/nodes-edge-remote-workers? Thanks!

**Comment by lyman9966** [comment]
@mburke5678 I will review the part of “Remote worker nodes on the network edge”

**Comment by lihongan** [comment]
The chapter “Remote worker nodes on the network edge” looks good to me. Thank you!

File: modules/nodes-nodes-jobs-creating-cron.adoc [hunk 1]
```diff
@@ -61,6 +62,52 @@ all subsequent executions will be suspended.
 <8> Job template. This is similar to the job example.
 <9> Sets a label for jobs spawned by this cron job.
 <10> The restart policy of the pod. This does not apply to the job controller.
+endif::openshift-rosa,openshift-dedicated[]
+ifdef::openshift-rosa,openshift-dedicated[]
```
Comment (mburke5678) [inline modules/nodes-nodes-jobs-creating-cron.adoc:66]: Example and call-outs copied from existing. Removed timezone

File: modules/nodes-scheduler-node-selectors-pod.adoc [hunk 2]
```diff
@@ -181,6 +182,7 @@ $ oc get nodes -l type=user-node,region=east
 NAME                          STATUS   ROLES    AGE   VERSION
 ip-10-0-142-25.ec2.internal   Ready    worker   17m   v1.26.0
 ----
+endif::openshift-rosa,openshift-dedicated[]
```
Comment (adellape) [inline modules/nodes-scheduler-node-selectors-pod.adoc:185]: This becomes a single-step (with substeps) procedure in the ROSA/OSD builds. Maybe there's a way to similarly make these individual `.` (non-sub) steps too, as in the other example.

**Review by adellape — COMMENTED** [review]
From a high-level view / sanity check perspective (per Slack review ping, and that there would be another follow-up to address other formatting/style issues), it looks like these are building correctly. I didn't find any broken areas in the rendered builds.

Just a few nits inline.

**Pull request merged**